src/quacc/recipes/mlp/elastic.py

```diff
@@ -0,0 +1,88 @@
+"""Elastic constants recipes for MLPs."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from quacc import flow
+from quacc.recipes.common.elastic import bulk_to_deformations_subflow
+from quacc.recipes.mlp.core import relax_job, static_job
+from quacc.wflow_tools.customizers import customize_funcs
+
+if TYPE_CHECKING:
+    from collections.abc import Callable
+    from typing import Any
+
+    from ase.atoms import Atoms
+
+    from quacc.types import ElasticSchema
+
+
+@flow
+def bulk_to_deformations_flow(
+    atoms: Atoms,
+    pre_relax: bool = True,
+    run_static: bool = False,
+    deform_kwargs: dict[str, Any] | None = None,
+    job_params: dict[str, dict[str, Any]] | None = None,
+    job_decorators: dict[str, Callable | None] | None = None,
+) -> ElasticSchema:
+    """
+    Workflow consisting of:
+
+    1. Deformed structures generation
+
+    2. Deformed structures relaxations
+        - name: "relax_job"
+        - job: [quacc.recipes.mlp.core.relax_job][]
+
+    3. Deformed structures statics (optional)
+        - name: "static_job"
+        - job: [quacc.recipes.mlp.core.static_job][]
```

Comment:
Both here and in `bulk_to_deformations_flow` in src/quacc/recipes/emt/elastic.py, the docstring is no longer complete with your introduced changes. The workflow now consists of up to 5 steps. This should be updated in both sections.